File: paquete_python/paquete_python/nodo.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node

class Clase_Nodo(Node):             # Clase que creamos heredando una clase de rclpy

    def __init__(self):
        super().__init__("nombre_nodo")     # Aquí insertamos el nombre del nodo
        
        self.get_logger().info("Baboleiro")

def main(args=None):
    rclpy.init(args=args)       # Inicia ROS2
    
    objeto_nodo = Clase_Nodo()  # Creamos el objeto del Nodo
    
    rclpy.shutdown()            # Cierra ROS2

if __name__ == '__main__':
    main()

File: paquete_cpp/src/nodo.cpp
#include "rclcpp/rclcpp.hpp"

class Clase_Nodo : public rclcpp::Node      // Clase que creamos heredando una clase de rclcpp
{
    public:
        Clase_Nodo() : rclcpp::Node("nombre_nodo_cpp")      // Aquí insertamos el nombre del nodo
        {
            RCLCPP_INFO(this->get_logger(), "Bamboleirlo");
            // Inicializar Variables
        }
    private:
        // Funciones Implementables
};

int main(int argc, char * argv[])
{
    rclcpp::init(argc, argv);       // Inicia ROS2

    rclcpp::spin(std::make_shared<Clase_Nodo>());   // Hace  que al llamar al nodo en la terminal, este no se cierre a menos que se pulse ctr+c

    rclcpp::shutdown();             // Cierra ROS2
    return 0;
}